### PR TITLE
added release notes, modified kylin and coprocessor upgrade guides

### PR DESCRIPTION
**Pull request by DingqianZhou**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: website/_docs/index.cn.md [hunk 1]
```diff
@@ -9,21 +9,31 @@ permalink: /cn/docs/index.html
 ------------  
 > Extreme OLAP Engine for Big Data
 
-Apache Kylin™是一个开源的分布式分析引擎，提供Hadoop之上的SQL查询接口及多维分析（OLAP）能力以支持超大规模数据，最初由eBay Inc.开发并贡献至开源社区。
+Apache Kylin™ 是一个开源 OLAP（On-Line Analytics Processing）分析引擎，针对数据库与亿万级记录提供亚秒级查询响应能力。2014 年由 eBay Inc.开发并贡献至开源社区。2015 年毕业于 Apache 顶级项目并迅速被全球数千家公司与组织应用于数据分析业务，同时也在 2015 与 2016 年赢得 InfoWorld Bossie Awards 年度最佳大数据开源工具。
```
Comment (shaofengshi) [inline website/_docs/index.cn.md]: "Online" is a word, not "On-Line", please check the terms: https://en.wikipedia.org/wiki/Online_analytical_processing

Comment (shaofengshi) [inline website/_docs/index.cn.md]: "Online" is a word, not "On-Line", please check the terms: https://en.wikipedia.org/wiki/Online_analytical_processing

File: website/download/index.cn.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ title: 下载
 
 #### v2.5.0
 - 这是2.4版本后的一个主要发布版本，包含了96 个以及各种改进。关于具体内容请查看[v2.5.0 正式发布](/blog/2018/09/20/release-v2.5.0/). 
-- [发布说明](/docs/release_notes.html) and [升级指南](/docs/howto/howto_upgrade.html)
+- [发布声明](/cn/docs/releases/kylin_release_notes.html) and [Kylin 升级](/cn/docs/howto/howto_upgrade.html)
```
Comment (shaofengshi) [inline website/download/index.cn.md]: We don't need to provide multiple language for release notes...

File: website/download/index.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ You can verify the download by following these [procedures](https://www.apache.o
 
 #### v2.5.0
 - This is a major release after 2.4, with 96 bug fixes and enhancement. Check the [v2.5.0 release announcement](/blog/2018/09/20/release-v2.5.0/) and the release notes. 
-- [Release notes](/docs/release_notes.html) and [upgrade guide](/docs/howto/howto_upgrade.html)
+- [Kylin Release Notes](/docs/releases/kylin_release_notes.html) and [Upgrade Kylin From Old Versions](/docs/howto/howto_upgrade.html)
```
Comment (shaofengshi) [inline website/download/index.md]: I think the "upgrade guide" in lower case is more friendly than "Upgrade Kylin From Old Versions" in hump...

File: website/_data/docs.yml [hunk 4]
```diff
@@ -78,6 +77,10 @@
   - howto/howto_use_cli
   - howto/howto_ldap_and_sso
   - howto/howto_use_beeline
-  - howto/howto_update_coprocessor
+  - howto/howto_upgrade_coprocessor
```
Comment (shaofengshi) [inline website/_data/docs.yml]: This change will change the URL endpoint. We hope the URL can be kept as there are many external references.

File: website/_docs/howto/howto_upgrade.md [hunk 5]
```diff
@@ -1,112 +1,146 @@
 ---
 layout: docs
-title:  Upgrade From Old Versions
+title:  Upgrade Kylin From Old Versions
```
Comment (shaofengshi) [inline website/_docs/howto/howto_upgrade.md]: Of course it is "Kylin", this looks redundant ...

File: website/_docs/howto/howto_upgrade.md [hunk 6]
```diff
@@ -1,112 +1,146 @@
...
 categories: howto
 permalink: /docs/howto/howto_upgrade.html
 since: v1.5.1
 ---
 
-Running as a Hadoop client, Apache Kylin's metadata and Cube data are persistended in Hadoop (HBase and HDFS), so the upgrade is relatively easy and user does not need worry about data loss. The upgrade can be performed in the following steps:
+Running as a Hadoop client, Apache Kylin's metadata and Cube data are persistent in Hadoop (HBase and HDFS), so the upgrade is relatively easy and user does not need worry about data loss. The upgrade can be performed in the following steps:
```
Comment (shaofengshi) [inline website/_docs/howto/howto_upgrade.md]: "persistent" is incorrect in this context, it is adjective, while here we need "be persisted" which is a passive voice;  (a typo in the previous version).

**Comment by DingqianZhou** [comment]
本 PR 先关闭。

**Comment by DingqianZhou** [comment]
I agree with @shaofengshi's comments, and in that way, this PR is not that necessary, thus I closed this PR.

**Pull request closed without merging**